src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -1,8 +1,8 @@
-package wolox.training.models;
+package wolox.training.repositories;
 
-import java.util.List;
 import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.Book;
 
 public interface BookRepository extends CrudRepository<Book, Long> {
-    List<Book> findByTitle(String title);
-}
\ No newline at end of file
+    Book findFirstByAuthor(String author);
```

Comment:
You could return the object wrapped inside an `Optional` to avoid returning null if the book is not found, like this: 
`Optional<Book>`

You can read more about it here: https://www.baeldung.com/java-optional 😄